vm/isolated/isolated.go

```diff
@@ -65,6 +71,9 @@ func ctor(env *vmimpl.Env) (vmimpl.Pool, error) {
 	if env.Debug && len(cfg.Targets) > 1 {
 		log.Logf(0, "limiting number of targets from %v to 1 in debug mode", len(cfg.Targets))
 		cfg.Targets = cfg.Targets[:1]
+		if len(cfg.Targets) > 1 && len(cfg.USBDevNums) > 1 {
```

Comment:
This check looks wrong, the line before this leaves only one target in `cfg.Targets`.